uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/helpers/ApiHelpersTrait.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+trait CRM_HRLeaveAndAbsences_ApiHelpersTrait {
+
+  private function callAPI($entity, $action, $params, $checkPermissions = false) {
```

Comment:
Since the idea is to enforce check_permission=true I think it would probably be better to swap things here a bit:
- The default value for `$checkPermissions` would be true
- `callAPIWithCheckPermissionsOn` would be renamed to `callAPIWithoutCheckPermissions` and it would pass false to `callAPI`